src/components/Swipeable.js

```diff
@@ -0,0 +1,221 @@
+        }).start()
+      }
+    }
+    })
+  }
+
+  onAnswered = (answer) => {
+    this.props.saveAnnotation(this.props.workflow.first_task, answer)
+    this.props.saveThenStartNewClassification()
+  }
+
+  render() {
+    const key = this.props.workflow.first_task //always just one task
+    const task = this.props.workflow.tasks[key]
+    const answers = reverse(task.answers)  //Yes is listed first in project, but we need No listed first (on left)
+    const imageSizeStyle = { width: this.props.subjectSizes.resizedWidth, height: this.props.subjectSizes.resizedHeight }
+    const swipeableSize = { width: this.props.subjectSizes.resizedWidth, height: this.props.subjectSizes.resizedHeight + 10 }
+    let pan = this.state.pan
+
+    let [translateX, translateY] = [pan.x, pan.y]
+
+    let rotate = pan.x.interpolate({inputRange: [-200, 0, 200], outputRange: ['-30deg', '0deg', '30deg']})
+    let opacityRight = pan.x.interpolate({inputRange: [0, 100, 200], outputRange: [0, .6, .8]})
+    let opacityLeft = pan.x.interpolate({inputRange: [-200, -100, 0], outputRange: [.8, .6, 0]})
+    let opacityRightText = pan.x.interpolate({inputRange: [0, 30], outputRange: [0, 1]})
+    let opacityLeftText = pan.x.interpolate({inputRange: [-30, 0], outputRange: [1, 0]})
+
+    let animatedCardStyles = {transform: [{translateX}, {translateY}, {rotate}]}
+    let leftOverlayStyle = {backgroundColor: leftOverlayColor, opacity: opacityLeft}
+    let rightOverlayStyle = {backgroundColor: rightOverlayColor, opacity: opacityRight}
+    let leftOverlayTextStyle = {opacity: opacityLeftText}
+    let rightOverlayTextStyle = {opacity: opacityRightText}
+
+    return (
+      <View style={[styles.container, {top: toTop + this.props.questionContainerHeight}]}>
+        <View style={styles.subjectContainer}>
+          <Animated.View
+            style={[styles.imageContainer, animatedCardStyles, swipeableSize]}
+            {...this._panResponder.panHandlers}>
+
+            <View style={swipeableSize}>
+              <SwipeSubject
+                inFront={true}
+                subject={this.props.subject}
+                subjectSizes={this.props.subjectSizes}
+                seenThisSession={this.props.seenThisSession}
+                setNextSubject={this.props.setNextSubject}
+              />
+              <Animated.View style={[styles.overlayContainer, leftOverlayStyle, imageSizeStyle]} />
+              <Animated.View style={[styles.overlayContainer, leftOverlayTextStyle, imageSizeStyle]}>
+                <StyledText additionalStyles={[styles.answerOverlayText]} text={ answers[0].label } />
+              </Animated.View>
+
+              <Animated.View style={[styles.overlayContainer, rightOverlayStyle, imageSizeStyle]} />
+              <Animated.View style={[styles.overlayContainer, rightOverlayTextStyle, imageSizeStyle]}>
+                <StyledText additionalStyles={[styles.answerOverlayText]} text={ answers[1].label } />
+              </Animated.View>
```

Comment:
Am I correct in assuming both the blue/red containers and also the yes/no text is _always_ present over the subject, but their visibility is determined by the swipe movement left or right?